projecting roi in point cloud

indukantdeo opened the pull request.

Commit:
projecting roi in point cloud

hari-sikchi reviewed (commented):
Can you elaborate on the running instructions in README?

indukantdeo commented:
pointcloudcapture is used for initial storing of point cloud from which we extract region of interest by converting the point cloud into image 
openniviewer is the executable created by the main.cpp in src 
to run openniviewer we need to pass the inputpointcloud as argument 
in the above code region interest is set such that it project jackal in the new point cloud
./openniViewer inputCloud1.pcd // for projecting jackal
./openniViewer inputCloud4.pcd // for projecting roof and measuring its distance which when measure for the measuring tape comes to be 1.76m

we used euclidean clustering on the projected pointcloud to filter the result but there is no such improvement on result

indukantdeo commented:
for running the code we just need to use ./openniVewer inputCloud1.pcd 
and we need to adjust the point cloud to view

hari-sikchi commented:
From what I gather most of the important code needed is in the src folder and outside. What is the use of build folder here. Also, there are a lot of pcd files lying around. If they are examples put them in the examples folder. You can remove the build folder and specify the compilation instruction and libraries required in its place in the README.  
Also, shift the folder in 
Methods--->Pointcloud_extraction-->Your work  
and also keep a proper README.md for your work.

indukantdeo commented:
I have already done those things i was trying to incorporate the multisense
file in the same folder i will be sending you the pr by today midnight

On 6 Aug 2017 19:34, "Harshit Sikchi" <notifications@github.com> wrote:

> From what I gather most of the important code needed is in the src folder
> and outside. What is the use of build folder here. Also, there are a lot of
> pcd files lying around. If they are examples put them in the examples
> folder. You can remove the build folder and specify the compilation
> instruction and libraries required in its place in the README.
> Also, shift the folder in
> Methods--->Pointcloud_extraction-->Your work
> and also keep a proper README.md for your work.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/hari-sikchi/RGBD-Vision/pull/2#issuecomment-320508960>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWhhb79eoywEOWK6lL5p3igTLoTTAJQXks5sVcfXgaJpZM4NkVfb>
> .
>

indukantdeo committed:
claeaned the package and files placed in appropriate folders
build folder removed
change in cmake to exclude build folder and creating executables folder
all inputcloud test samples are now placed in data folder

indukantdeo committed:
build removed

indukantdeo committed:
added multisense_pointcloud

indukantdeo committed:
multisense point cloud added

hari-sikchi commented:
Everything looks good! Just move the kinect and multisense folder inside the methods folder and you are good to go.

indukantdeo committed:
kinect and multisense moved to methods folder

indukantdeo committed:
multisense and kinect_pointcloud removed from root

indukantdeo committed:
Update readme.txt
